src/common/call-async-step-function.ts

```diff
@@ -0,0 +1,44 @@
+import type { MarkActivityAsReportedInput } from "./model";
+import {
+  SFNClient,
+  StartExecutionCommand,
+  StartExecutionInput,
+  StartExecutionOutput,
+} from "@aws-sdk/client-sfn";
+
+export async function callAsyncStepFunction(
+  stateMachineArn: string,
+  input: MarkActivityAsReportedInput
+): Promise<string> {
+  let response: StartExecutionOutput;
+  try {
+    console.log("Starting state machine execution.");
+    response = await startASyncStepFunction(stateMachineArn, input);
+  } catch (error: unknown) {
+    console.error("Failed to start Report Suspicious Activity state machine.", {
+      stepFunctionArn: stateMachineArn,
+    });
+    console.error("Reason Detail: ", JSON.stringify(error));
```

Comment:
Do you know if the contents of this error could contain the inputs to the step function? We have sensitive data in there (eg. email) and we shouldn't be logging that.

Reply:
Addressed - Thanks Alex, as it is Lambda, it can. so rather than stringify all error structure, I parse and only log the error message